### DISCUSSION TITLE
having trouble running xpstopdf after installing libgxps

### DISCUSSION THREAD

**Discussion by andrewjyee** [Homebrew/discussions#3407] [Getting started] [answered]
I'm running brew on Mac OS 12.4 on a 16" Macbook Pro.  I was able to install `libgxps` using `brew install`.  However, when I tried to run `xpstopdf`, I'm getting `zsh: command not found: libgpxs`
I did `echo $PATH` and it shows

`/opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Library/Apple/usr/bin`

I see `libgxps` in `/opt/homebrew/Cellar` but not elsewhere

Apologies for the naiveness of these questions.

**Comment by carlocab** [comment, marked as answer]
When you installed `libgxps`, you should have seen this message:
```
==> Caveats
`ghostscript` now installs a conflicting libgxps.dylib.
You may need to `brew unlink libgxps` if you have both installed.

libgxps is keg-only, which means it was not symlinked into /opt/homebrew,
because it conflicts with `ghostscript`.
```
You can also see it again with `brew info libgxps`. You can read the [FAQ](https://docs.brew.sh/FAQ#what-does-keg-only-mean) for more information about what it means to be keg-only.

You can add `libgxps` to your `PATH` with
```bash
export PATH="$(brew --prefix libgxps)/bin:$PATH"
```
You may wish to consider adding this to your `.zprofile` (or perhaps `.zshrc`) so you don't need to do it every time you open a terminal.

**Reply by andrewjyee** [reply]
Ah this answered my question.  I guess I should have paid more attention to the keg-only caveats!

**Reply by carlocab** [reply]
That would help. `brew` can tend to print too many things at once (especially with `brew install`), but we try hard to make sure that the most important messages are shown (possibly for a second time) last.

Checking the last few lines of output from `brew` for things like `caveats` would probably be enough.
